gnmi_server/server.go

```diff
@@ -274,6 +277,26 @@ func (s *Server) checkEncodingAndModel(encoding gnmipb.Encoding, models []*gnmip
 	return nil
 }
 
+func ParseOrigin(origin string, paths []*gnmipb.Path) (string, error) {
```

Comment:
>origin [](http://example.com/codeflow?start=17&length=6)

why a parse function has an extra input parameter `origin`? Could you add a function level comment?

Reply:
I will fix this.